api/apps/api/test/projects/projects.fixtures.ts

```diff
@@ -547,3 +640,78 @@ export const getFixtures = async () => {
     },
   };
 };
+
+/**
+ * @debt Copy of same function from api/apps/geoprocessing/test/integration/cloning/fixtures.ts
+ */
+export function GivenOrganizationExists(
+  em: EntityManager,
+  organizationId: string,
+) {
+  return em
+    .createQueryBuilder()
+    .insert()
+    .into(`organizations`)
+    .values({
+      id: organizationId,
+      name: `test organization - ${organizationId}`,
+    })
+    .execute();
+}
+
+/**
+ * @debt Copy of same function from api/apps/geoprocessing/test/integration/cloning/fixtures.ts
+ */
+export async function GivenProjectExistsV3(
+  em: EntityManager,
+  projectId: string,
+  organizationId: string,
+  projectData: Record<string, any> = {},
+  costSurfaceId = v4(),
+) {
+  await GivenOrganizationExists(em, organizationId);
+
+  const insertResult = await em
+    .createQueryBuilder()
+    .insert()
+    .into(`projects`)
+    .values({
+      id: projectId,
+      name: `test project - ${projectId}`,
+      organizationId,
+      planningUnitGridShape: PlanningUnitGridShape.Square,
+      ...projectData,
+    })
+    .execute();
+
+  await GivenDefaultCostSurfaceForProject(em, projectId, costSurfaceId);
+
+  return insertResult;
+}
+
+/**
+ * @debt Copy of same function from api/apps/geoprocessing/test/integration/cloning/fixtures.ts
+ */
+async function GivenDefaultCostSurfaceForProject(
+  em: EntityManager,
+  projectId: string,
+  id: string,
+  name?: string,
+) {
+  const nameForCostSurface = name || projectId;
+  return em
+    .createQueryBuilder()
+    .insert()
+    .into(`cost_surfaces`)
+    .values({
+      id,
+      name: `${
+        nameForCostSurface ? nameForCostSurface + ' - ' : ''
+      }Default Cost Surface`,
+      projectId: projectId,
+      min: 0,
+      max: 0,
+      isDefault: true,
+    })
+    .execute();
+}
```

Comment:
Is there any special reason why you're not importing these functions instead? Not a big deal, but it could easier to refactor down the line importing them; having a copy might instill doubtfulness as to whether these are the exact same functions or they defined separately because of any possible issue.

Reply:
yep, it's ugly. I'd have to move these fixtures to somewhere under `api/libs` though, as the original code is in `geoprocessing` and here I'm using it in `api`, and when we build the OCI images for `api` test runners in CI we delete the whole tree of `geoprocessing`.

Indeed I thought about tidying up some common functions used in tests somewhere, but given that there is a lot of duplication and kinduplication (kind of duplication but not quite, with minor divergences here and there, and so on) already in our tests monster, I thought I'd just give up at this stage.

Do you have suggestions on how to handle this without much moving around of stuff?

Reply:
Oh, yeah... the api/geo split. Yeah, I see what you mean now.
I agree that other than having a module on libs with common methods for testing on both apis, would be the way to go, but probably not worthy right now. To do it properly, would require scouring the tests quite a bit.
So, for now, doing it like this and tagging it as debt (maybe with some of indicator like "candidate for common tests utils grouping" or something) should be enough.

Reply:
agree. I've added a bit more detail to the `@debt` comments, and also added these to the geoprocessing functions from which I copied the api ones, so that the debt is clear on both sides.

Reply:
acf7ff46a26c9506ac7c130c0371e7793434e192